.github/workflows/Dockerfile.ssr

```diff
@@ -11,8 +11,8 @@ COPY .nuxt ./.nuxt
 COPY static ./static
 
 # Install runtime dependencies
-RUN yarn install --production --no-lockfile
+RUN npm install --production --no-save --no-audit
```

Comment:
Why not go with `npm ci` in that case? :)

Reply:
Same as above